AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -22,7 +22,9 @@ struct SessionHeaderView: View {
             HStack {
                 dateAndTime
                 Spacer()
-                actionsMenu
+                ZStack {
```

Comment:
You use ZStack here but there's just one thing inside of it, so you don't need the ZStack here :D